File: libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/NavigationWalkingOptionsTest.java
package com.mapbox.services.android.navigation.v5.navigation;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class NavigationWalkingOptionsTest {

  @Test
  public void alleyBias_walkingOptionSet() {
    NavigationWalkingOptions options = NavigationWalkingOptions.builder().alleyBias(0.7).build();

    assertEquals(Double.valueOf(0.7), options.getWalkingOptions().alleyBias());
  }

  @Test
  public void walkwayBias_walkingOptionSet() {
    NavigationWalkingOptions options = NavigationWalkingOptions.builder().walkwayBias(0.8).build();

    assertEquals(Double.valueOf(0.8), options.getWalkingOptions().walkwayBias());
  }

  @Test
  public void walkingSpeed_walkingOptionSet() {
    NavigationWalkingOptions options = NavigationWalkingOptions.builder().walkingSpeed(2.0).build();

    assertEquals(Double.valueOf(2.0), options.getWalkingOptions().walkingSpeed());
  }
}

File: libnavigation-base/src/test/java/com.mapbox.navigation.base/route/NavigationWalkingOptionsTest.kt
package com.mapbox.navigation.base.route

import org.junit.Assert.assertEquals
import org.junit.Test

class NavigationWalkingOptionsTest {

    @Test
    fun alleyBias_walkingOptionSet() {
        val options = NavigationWalkingOptions.builder().alleyBias(0.7).build()

        assertEquals(java.lang.Double.valueOf(0.7), options.walkingOptions.alleyBias())
    }

    @Test
    fun walkwayBias_walkingOptionSet() {
        val options = NavigationWalkingOptions.builder().walkwayBias(0.8).build()

        assertEquals(java.lang.Double.valueOf(0.8), options.walkingOptions.walkwayBias())
    }

    @Test
    fun walkingSpeed_walkingOptionSet() {
        val options = NavigationWalkingOptions.builder().walkingSpeed(2.0).build()

        assertEquals(java.lang.Double.valueOf(2.0), options.walkingOptions.walkingSpeed())
    }
}
